project-management/src/main/java/com/smartagilify/projectmanagement/services/wbs/WBSServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.smartagilify.projectmanagement.services.wbs;
+
+import com.smartagilify.core.enumerations.EN_STATE;
+import com.smartagilify.core.services.BaseService;
+import com.smartagilify.projectmanagement.exceptions.ApplicationException;
+import com.smartagilify.projectmanagement.mappers.wbs.WBSMapper;
+import com.smartagilify.projectmanagement.model.dtos.WBSDTO;
+import com.smartagilify.projectmanagement.model.entities.Project;
+import com.smartagilify.projectmanagement.model.entities.WBS;
+import com.smartagilify.projectmanagement.repositories.project.ProjectRepository;
+import com.smartagilify.projectmanagement.repositories.wbs.WBSRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class WBSServiceImpl extends BaseService<WBS, WBSMapper, WBSDTO> implements WBSService{
+
+    private final WBSRepository wbsRepository;
```

Comment:
cast `jpaRepository` instead of injecting `WBSRepository` again.
`jpaRepository` is `WBSRepository`